scripts/zones/Chamber_of_Oracles/mobs/Blizzard_Wyvern.lua

```diff
@@ -0,0 +1,21 @@
+-----------------------------------
+-- Area: Chamber of Oracles
+--  Mob: Blizzard Wyvern
+-- KSNM: Eye of the Storm
+-----------------------------------
+require("scripts/globals/status")
+-----------------------------------
+function onMobSpawn(mob)
+    mob:setMobMod(tpz.mobMod.SIGHT_RANGE, 17)
+end
+
+function onMobEngaged(mob, target)
+    mob:useMobAbility(815)
+end
+
+function onMobFight(mob, target)
+    mob:setMod(tpz.mod.REGAIN, 100)
```

Comment:
`onMobFight` or `onMobEngaged`?

Reply:
I'm not sure which is right - they basically use it as soon as they see you. if onmobfight is better, will do there. it works as is in terms of testing, but rather do it in the right place if they are different.

Reply:
onMobEngaged is when the mob aggros (and also when it reaggros); onMobFight is constantly - I believe it's every attack round

Reply:
oh yes, i remember now. most definitely how i have it, when it aggros in onmobengaged